.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
```

Comment:
- [ ] Great job on adding the linters.yml file 👏 . but as requested by the previous reviewer, you did not finish with your linter configuration because it doesn't run, please let's follow [these instructions](https://github.com/microverseinc/linters-config/tree/master/html-css) on how to set up the linters correctly for this repository.
  - You should copy [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc) to the root directory of your project, and
  - You should copy [.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css/.stylelintrc.json) to the root directory of your project.

Reply:
@BenMukebo That's strange! It did run thrice and now it is no longer running

Reply:
@BenMukebo How do I resolve conflicts in my repository?

Reply:
Hi @j-tee, let me send you something that can help you to resolve the conflict